mlflow/models/output_parsers.py

```diff
@@ -0,0 +1,60 @@
+from dataclasses import asdict
+from typing import Any, Dict
+
+from mlflow.models.rag_signatures import (
+    ChainCompletionChoice,
+    ChatCompletionResponse,
+    Message,
+    StringResponse,
+)
+from mlflow.utils.annotations import experimental
+
+try:
+    from langchain_core.output_parsers.transform import BaseTransformOutputParser
+
+    @experimental
+    class ChatCompletionsOutputParser(BaseTransformOutputParser[Dict[str, Any]]):
```

Comment:
Can we add test coverage for these parsers? Else, we won't know whether they'll continue to work as LangChain releases new versions